lib/sendMetrics/index.js

```diff
@@ -0,0 +1,34 @@
+const SDC = require('@fiverr/statsd-client');
+
+function send({ statsdClient, dataFromApi }) {
+    const statsd = new SDC({ host: statsdClient.host, port: statsdClient.port });
+    const data = prepareDataFromApi(dataFromApi);
+    data.forEach((permutation) => {
+        statsd.gauge(buildStatsdMetric(permutation.page, permutation.strategy, permutation.category), permutation.score);
+    });
+}
+
+function prepareDataFromApi(dataFromApi) {
+    const data = [];
+    dataFromApi.forEach((arr) => {
+        arr.forEach((url) => {
+            const pageName = url.pageName.split(' ').join('').toLowerCase();
```

Comment:
I suggest replacing with underscore will be more readable
`productlistings` => `product_listings`
`userbalance` => `user_balance`
 
```suggestion
            const pageName = url.pageName.replace(/\s+/g, '_').toLowerCase();
```

Reply:
will do